src/widgets/__tests__/index.tests.ts

```diff
@@ -0,0 +1,172 @@
+import { PlacesInstance } from 'places.js';
+import * as widgets from '../';
+import { Widget } from '../../types';
+
+// @TODO: move to typedObject once this no longer needs to be polyfilled
```

Comment:
Not sure this needs to be a TODO since it depends on the language support and not on us. Wdyt?

Reply:
what I meant is that this utility can't be in typedObject, since our polyfill test will fail if source files use object.entries, which is why I put the utility only in the test. I can remove the comment if you don't think it's important

Reply:
Then you might change the comment from "TODO" to "It's not in the source but in this test because .."